plaso/filters/path_filter.py

```diff
@@ -14,6 +14,8 @@
 
 from plaso.lib import py2to3
 
+# pylint: disable=missing-type-doc,missing-return-type-doc
```

Comment:
Please make these local overrides instead of overrides for the whole module

Reply:
Done